db/migrate/20160324174218_create_container_deployment_nodes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerDeploymentNodes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_nodes do |t|
+      t.string :ip_or_hostname
+      t.string :name
+      t.text :labels
+      t.belongs_to :container_deployment, :type => :bigint
+      t.belongs_to :vm,                   :type => :bigint
+      t.text :docker_storage_devs, :array => true, :default => []
+      t.string :docker_storage_data_size
```

Comment:
@detiber I mentioned in a thread, we were going to come up with a default VG name, something like `docker-storage` or `miq-docker-storage` anyway it will be in the main ManageIQ settings (always the same for all the installs).